### PR TITLE
Tweaks cult pylons

### PR DESCRIPTION
**Pull request by ChangelingRain**
Pylons now process rapidly and heal more often:

Healing is about the same per second, but is faster on human cultists that have both brute AND burn damage.

The faster processing means the healing might be slightly faster/more per second overall due to checking if it can heal more often, but it's still not a lot of healing at all.

Pylons now generate a list of valid turfs to convert every 5 seconds, which means they'll properly update if turfs get removed/damaged near them but has the downside of possibly converting turfs outside the hideout if it's too close to the entrance.

### CODE REVIEW

**Comment by Robustin** [comment]
The whole "not updating the turfs" list was intentional since you don't have "CULT BASE PLZ RAID US" tiles sitting outside your false wall or whatever. I would really be sad seeing cult bases get out'd by a decorative aspect of cult structures. 

As for the healing change, it was always faster on brute/burn combo but what is your rough estimate of brute healing per 5 seconds now? I'm not opposed, the pylons, even when stacked, are a long way from having any issues (until someone finally abuses them to manifest an army). Just curious.

**Comment by ChangelingRain** [comment]
2 brute healing per 5 sec, same on constructs. Both brute and burn is 4 per 5 sec total.

The turf thing was frankly weird and kind of unpredictable for players because it didn't actually respect view; air goes through falsewalls so it'd actually already go through those unless you made the falsewall *after* the pylon.
Any sort of update thing will have the possibility of dropping a turf where you don't want it, but the previous solution was actually more unpredictable for players because most players won't know *how* it works, only that it converts tiles to cult tiles.

**Comment by Robustin** [comment]
Air needs to stop going through falsewalls then. Just because we can slide them out of them way doesn't mean the wall isn't (mostly) airtight when shut.

File: code/game/gamemodes/cult/cult_structures.dm [hunk 1]
```diff
@@ -110,14 +108,20 @@
...
-							M.adjustHealth(-2)
-		if(corruption.len)
-			var/turf/T = pick_n_take(corruption)
-			corruption -= T
+							M.adjustHealth(-1)
+			CHECK_TICK
+	if(last_corrupt <= world.time)
+		var/list/validturfs = list()
+		for(var/T in circleviewturfs(src, 5))
 			if(istype(T, /turf/closed) || istype(T, /turf/open/floor/engine/cult) || istype(T, /turf/open/space) || istype(T, /turf/open/floor/plating/lava))
```
Comment (ChangelingRain) [inline code/game/gamemodes/cult/cult_structures.dm]: It only runs every 5 seconds, 10 if it found no turfs on last iteration, so it ~should~ be okay

Comment (RemieRichards) [inline code/game/gamemodes/cult/cult_structures.dm]: Be prepared to up the time delay if necessary.

**Pull request merged**